src/main/java/frc/robot/commands/led/Gaming.java

```diff
@@ -0,0 +1,26 @@
+package frc.robot.commands.led;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.led.LED;
+
+public class Gaming extends CommandBase {
+
+    private LED led;
+    private int jumps;
```

Comment:
i dont understand what is jumps, but i'll recommand you using `IntSupplier `for more modular code.